### ISSUE TITLE
The file FlutterSearchableOptionContributor.java should be curated to make sure that there are no lines of dead code.

### ISSUE THREAD

**Issue by jwren** [flutter/flutter-intellij#7857]
At the top of this file: `// Note: update this file when updating FlutterSettingsConfigurable.`

**Comment by pq** [comment]
@jwren: do you know what update you were trying to track here?

**Comment by jwren** [comment]
@pq See https://github.com/flutter/flutter-intellij/blob/master/flutter-idea/src/io/flutter/sdk/FlutterSearchableOptionContributor.java -- we should curate this file content to remove dead code.

**Comment by pq** [comment]
I still don't understand the original issue description. Did you remove some code from `FlutterSettingsConfigurable` and this was a reminder to update the contributor?

Anyway, looking for dead code is always a good idea!

**Comment by jwren** [comment]
Yes, this is a task reminder to remove the dead lines of code in https://github.com/flutter/flutter-intellij/blob/master/flutter-idea/src/io/flutter/sdk/FlutterSearchableOptionContributor.java.

Updating the issue to make it clear
